typescript/commands/settings.py

```diff
@@ -0,0 +1,20 @@
+import sublime_plugin
+
+from ..libs.global_vars import *
+from ..libs import cli, logger
+import os
+
+class TypescriptOpenPluginDefaultSettingFile(sublime_plugin.WindowCommand):
```

Comment:
Though Sublime transforms this camel casing to snake casing, which means "TypeScript" will be transformed to "type_script" when referred to in sublime-menu files.